### ISSUE TITLE
Bug report - [App store not available- FireHD]

### ISSUE THREAD

**Issue by darkoverlordofdata** [CypherpunkArmory/UserLAnd#681]
# Popup says "Continue to Amazon Appstore? - this app is trying to open a link with an external market that is not available on your device". I can Cancel or Continue. 

# Steps to reproduce the behavior:
Start Usrland app. 
Select Ubuntu from Apps 
Enter username & passwords & continue
Select vnc & contiunue
It downloads & verifies assets, sets up filesystem
Then I get the popup.
I select continue, it brings up the appstore & nothing is selected.
I back out
Go to Sessions, select Ubuntu.
Same popup. same appstore, nothing is selected. I back out.
Nothing happens.

# Screenshots

# Device Information

Device: [FireHD 10 (7th Generation].  
Android Version: [5.5.0.0 – November 2017, only for Fire HD 10 (2017) with hands-free Alexa].  
UserLAnd Version [2.3.8].  
Whether the device is rooted: No.  
Whether the device is running LineageOS: No 

# Additional context
I have termux installed and running with a Gui. Never had an issue, except its tedius to get it up and running.

**Comment by darkoverlordofdata** [comment]
Dude, your rules are onerous.

**Comment by corbinlc** [comment]
It is a bot that one of our team members wrote.  We will improve it over time.  We have actually never tried this on a Kindle.  That is in our roadmap.

**Comment by BirdMachine** [comment]
Well, Just got my Fire 7 5th gen rooted, running a nexus ROM (https://forum.xda-developers.com/amazon-fire/orig-development/rom-fire-nexus-rom-lmy49f-t3300714) and I've got userLAnd installed via FDroid. Haven't had any luck but I only just started tinkering maybe an hour ago? Either way, I'm interested in helping out however/if I can with testing, debugging, and all of that wonderful stuff. Is there an open ticket yet for me to start looking into and working with, or better to just clone and dig in per the Contribution guide?
